conf/application.conf

```diff
@@ -69,11 +75,12 @@ mongodb {
 }
 
 urls {
-  login         = "http://localhost:9949/auth-login-stub/gg-sign-in"
-  loginContinue = "http://localhost:30029/members-protections-and-enhancements"
-  signOut       = "http://localhost:9025/gg/sign-out"
-  psaOverview   = "http://localhost:8204/manage-pension-schemes/overview"
-  pspDashboard  = "http://localhost:8204/manage-pension-schemes/dashboard"
+  login             = "http://localhost:9949/auth-login-stub/gg-sign-in"
+  loginContinue     = "http://localhost:30029/members-protections-and-enhancements"
+  signOut           = "http://localhost:9025/gg/sign-out"
+  psaOverview       = "http://localhost:8204/manage-pension-schemes/overview"
+  pspDashboard      = "http://localhost:8204/manage-pension-schemes/dashboard"
+  checkAndRetrieve  = "members-protections-and-enhancements/check-and-retrieve"
```

Comment:
It looks like the context for the backend is `/members-protections-enhancements` without the 'and' https://github.com/hmrc/members-protections-enhancements/blob/main/conf/prod.routes